db/migrate/20170125141953_update_description_in_miq_alert_status.rb

```diff
@@ -0,0 +1,19 @@
+class UpdateDescriptionInMiqAlertStatus < ActiveRecord::Migration[5.0]
+  class MiqAlertStatus < ActiveRecord::Base
+    belongs_to :miq_alert, :class_name => "UpdateDescriptionInMiqAlertStatus::MiqAlert"
+  end
+
+  class MiqAlert < ActiveRecord::Base
+    has_many :miq_alert_statuses, :dependent => :destroy, :class_name => "UpdateDescriptionInMiqAlertStatus::MiqAlertStatus"
+  end
+
+  def up
+    say_with_time("update description in miq alert statuses") do
+      miq_alerts = Arel::Table.new('miq_alerts')
+      miq_alert_statuses = Arel::Table.new('miq_alert_statuses')
+      join_sql = miq_alerts.project(miq_alerts[:description])
+                           .where(miq_alerts[:id].eq(miq_alert_statuses[:miq_alert_id])).to_sql
+      MiqAlertStatus.update_all("description = (#{join_sql})")
```

Comment:
Heh, I would have done sql instead of areal but it looks good.

```sql
update miq_alert_statuses
set description = (
  select miq_alerts.description
  from miq_alerts
  where miq_alerts.id = miq_alert_statuses.miq_alert_id
)
```

LGTM :+1: